### DISCUSSION TITLE
子农场内如何获得高收益要点的看法分享

### DISCUSSION THREAD

**Discussion by yancanlou** [famland/website#98] [Show and tell]
关于农场内获取高收益大家都有自己的理解，白皮书也做了大篇幅的解读，这里要分享的是个人的几个看法。

先看这个关于耕种收益方面白皮书给出的公式为：

耕种收益=当前收益时长（分钟）×（（基本田做市资金价值+原始田做市资金价值）×肥力值加成系数+野田做市资金价值）×收取收益时农场实时的年化APY/525600
做市资金价值=存入的USDT数量+存入的YL数量×YL实时价格

从上述公式算法中获知：
主要因素为，本次做市时长，做市资金价值，加成系数，实时年化APY四大块组成，详细如下。

1.耕种时长对同年化下获取收益成正比？实际时间长与短都在一个元宇宙内，长则收的次数少积多，短则收的次数多积小。时间对于每一块田是公平的不存在正比关系，核心区别在于收取的时间点对应的实时APY不同。由于算法是按分钟计，其取值影响计算，极端情况太频繁少于一分钟收取很显然永远没有收益，是对于分钟的浪费。
2.基本田做市资金价值的情况，如果你的基本田是满额500U，即250U+存入的YL数量×YL实时价格。此处可以看出当YL价格低时做市，此田在YL价格高时将获得比其它田更高的资金价值，收益就更高。
3.原始田做市资金价值情况与基本田相同，等于额外增加了做市资金价值，但原始田属于生态助益性质更多就不多介绍。
4.肥力值加成系数，这个系数中最重要的一点就是NFT的栽培，其栽培打开了系数通道，获得提升的同时使得肥力值的（采用每2万渐增法加成阶级）增加有了升值空间。
5.野田做市资金价值，想要获得野田做市资金价值的加成，就要充分利用野田的进出无影响的特点，在农田上加入野田做市资金价值就可以，因为基本田的做市价值是有上限的，所以野田的做市价值大小是农场内获得更多收益分配的主要方式（野田做市上限受限于肥力值阶段2万，所以这种情况下玩家有能力最终还是要培养一块满额田的）。
6.不同的时间点对应的实时（分钟）年化APY变化不一，而APY是按5分钟进行更新的（很重要）。所以农场实时（分钟）年化APY高时收取有利。而更为重要的是，农场内的年化APY是依日收益预期最大M值来计算，当其它田都去结算YL收益时，本农场内的M值的就会变小，则当日最后收取的人，计算出的年化APY会变得最高，其分发的收益最大。

影响利弊分析：
1.YL价格低时做市加成因数变大，当YL价格上涨时收取收益必然增加，收取收益时应当于YL价格上涨时进行为佳。
2.从APY影响因素又得知，YL价格上涨又直接提升了APY，所以收益增加点就是YL价格增值，增值越大APY越高，所获YL越多。
3.满额做市基本田在非满额作市的基本田和野田组合农场内，终会被排斥。因其不旦拉高了农场的M值，使得该农场的APY值降低，从而间接使其它田每次收取的YL变少，而其本身收取却占有大头。
4.同理与有原始田的情况也是基本相同，所以发展的趋势是，只有相近的做市资金价值的才适合组合在一起，哪怕投入本金相同，但由于在YL价格的不同阶段进入，其共享值相近，但其做市资金价值则不相近，无法达成最具效率共识。
5.肥力值加成系数NFT的栽培是提升收益，获更多YL的必要手段，其丰富了共识层次，又给农场组合的分化提供了动力与需求。
6.野田做市资金价值的配置是农田玩家很有必要贡献增量，并提升YL收益的操作。这个操作要求就是备着点子弹，最好是在YL价格低于基准价对半再买入YL（或者本身备好YL）做市野田，以获得更多YL后再卖回（当池子稳定性好时可以一直全仓持有，以获得最大收益），反复操作提升共享值上限及增量。
7.共享值高低出身差异不是受农场欢迎的接收条件，只有具备增量值高的才能弥补做市资金价值差异被农场接收，想获得高年化，最好的方法是必须进高年化的农场，才能最大程度的提升增量，并不断地争取收益扩大。那些只会复投但不进行增量的农田，是每个农场中的驻虫存在，因其光追求系数加成的个人收益，却不提供农场流速贡献。
8.所谓的获得更多YL,实际上是在抢占农场内其它田的分发池份额，各农场间的杀手级收益在于以静制动，因为要关注的点是：APY是每5分钟更新一次(M值变化)，而分发池是间隔一小时多点增加一次，而这是一个复杂的博弃，可见选择独立建农场越早越好。
9.因增量的增长受限于共享值上限，其上限又受限于基本田地块的收益积累。所以早期农场（特别是1号农场先发培值的原30块里的高上限田），由于满级基本田，加成系数高的田，受其共享值上限增长快，将在生态内形成强大的马太效应。后建的农场想要获得初期增量起步快，必须要有从其它农场挖墙角而来大增量田，这需要社区的互动与市场的活跃，买田或拉人是重要且不可或缺的起步条件。在此正生长螺旋下，农场建成的首发要素是增量，否则一步落后，步步落后，这是条要长期竞赛的路。
10. 对于大多数的小户来讲只能在早期全力打造一块联盟农场，到后期才能有生存空间，所以大众玩家仍然是越早进入越有利。而土豪靠人脉跟资金买高增量的田才能实现直接小康起步。
11. 如果将来推出合田，设定增量增长的最高上限限定后，那些没有参与合田的升级农场后期几乎无竞争力，而和田玉的获得随机性将成为多数散户的生存之道，其农场收益低基本只能靠生态分红来维持了。
12.因为原生农场并没有保底占比流速，必将成为荒田沙漠。其受限于培养上限值的基本田收益低，但出路与自救也只能通过野田做市来尽可能的补上增量，为随时建农场或加入其它农场做下充足功课。

**Comment by cryptosoul-vioiv** [comment]
相当不错的分析。
唯一的瑕疵是遗漏了一个因素：共享值上限。
第3点，满额基本田收取的收益确实比非满额基本田要多，同时意味着其共享值上限也大，可提升的增量共享值亦大，更能提升流速。

贡献越大，收益越高，合情合理！

**Reply by yancanlou** [reply]
谢谢提点啊，不足之处多指正。

**Comment by yancanlou** [comment]
补充一点，有人问怎么提高APY？因为APY是收益的关键指标，其实上面的看法都有描述，也在群说过了。就是农场内的APY是由散博弈或者强管理出来的，军心散的杂农场依靠博弈来达到M值降低，临时的提升APY，提升个人收益。
而一致行动的农场则通过有执行力的交替轮收管理来降低M值，保证了APY的提升，从而提升每个人的收益。
当然上面都是建立在农场有足额分发池与稳定流速基础之上，这样才能实现提升后的APY稳定，最终调整到农场最优APY。
所有没有建立共识的农场很难经营，自主经营变成一种选择。而考虑团队的力量，强共识管理的农场才是最终赢家，这点到目前为止，通过APY这一稳定指标没有看到，APY现在都是突高性的，大家要努力。

**Reply by yancanlou** [reply]
从1号农场的兴起与衰败，验证了团队管理与农场主作用的关键因素，1号农场有着天然先发优势，一度分发池积累稳定在2万多yl,由于大开大合，一下体量太大，不选择的进入了许多共识不强也不愿合作的博弈者，都寄希望于他人先收，自己得利。始终无法进行整体轮动降低M值，最终APY提不起来，增量增长缓慢，流速占比慢慢降低，最终成员纷纷离场。
而单人占比高达80-90%的农场，农场主不论愿意不愿意必须先苦后甜，在日常主动于低APY时收取降低M值，让利给10-20%的农田，最终增量提速，得到共同发展。
这给其它的农场提供了一个参考，农场里没有合作无法共赢。
非分类没有共识，否则与原生农场无异。
按现规则优化后，3号农场的成长性减半。而2号农场已按当下算法整全了生态内最优资源配置，如果不团结合作，最终会被什么农场超越？
